### DISCUSSION TITLE
What is the purpose of opentelemetry::global::set_text_map_propagator

### DISCUSSION THREAD

**Discussion by gliderkite** [open-telemetry/opentelemetry-rust#1105] [Q&A] [answered]
What is thew purpose of [opentelemetry::global::set_text_map_propagator](https://docs.rs/opentelemetry/latest/opentelemetry/global/fn.set_text_map_propagator.html)?

I can't find any exhausting documentation about it, and I was able to setup traces propagation between multiple services (following [this answer](https://stackoverflow.com/a/73751405/995246)) without the need to set any [TextMapPropagator](https://docs.rs/opentelemetry/latest/opentelemetry/propagation/trait.TextMapPropagator.html).

**Comment by jtescher** [comment, marked as answer]
The propagation API spec should have the clearest answers for the purpose and requirements. Here's the specific requirement for setting a global propagator https://github.com/open-telemetry/opentelemetry-specification/blob/v1.22.0/specification/context/api-propagators.md#set-global-propagator

**Comment by gliderkite** [comment]
Perfect, thanks!
I link to those docs in the Rust API docs would be useful, if not having a summary directly in there.
